pkgs/applications/misc/keystore-explorer/default.nix

```diff
@@ -0,0 +1,28 @@
+{ fetchzip, stdenv, jre }:
+
+stdenv.mkDerivation rec {
+  version = "5.4.4";
+  name = "keystore-explorer-${version}";
+  src = fetchzip {
+    url = "https://github.com/kaikramer/keystore-explorer/releases/download/v${version}/kse-544.zip";
+    sha256 = "01kpa8g6p6vcqq9y70w5bm8jbw4kp55pbywj2zrhgjibrhgjqi0b";
+  };
+
+  buildInputs = [ jre ];
+
+  installPhase = ''
+    substituteInPlace ./kse.sh --replace "java -jar" "${jre}/bin/java -jar"
+    mkdir -p $out/bin
+    cp -R ./* $out/
```

Comment:
Haven't built it, but it looks like you are adding all of the source code to $out - are you sure that's necessary?